### PR TITLE
Initial version of email MFA doc

### PR DESCRIPTION
**Pull request by aaguiarz**
https://docs-content-staging-pr-7148.herokuapp.com/docs/multifactor-authentication/factors/email

### CODE REVIEW

File: articles/multifactor-authentication/factors/email.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+After the login step, users will be presented with the most secure enabled factor. If they select 'Other login options', and then pick Email, they will be sent an email with a six-digit code that they will need to enter to complete the authentication flow.
+
+![Email End User 1](/media/articles/multifactor-authentication/mfa-email.png)
+
+After Email MFA is enabled, all new or existing users from Database Connections that do not have verified emails will get prompted to verify their emails after their log in. They will get a code in ther email they'll need to enter in the login page to continue.
+
+## Administrative setup
+
+In order to set up Email, you need to able the Email factor in the Dashboard.
```
Comment (santiagoaguiar) [inline articles/multifactor-authentication/factors/email.md]:
```suggestion
In order to set up Email, you need to enable the Email factor in the Dashboard.
```

File: articles/multifactor-authentication/factors/email.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+![MFA Email Settings](/media/articles/multifactor-authentication/email-settings.png)
+
+[Auth0 provides a test email provider](/email) but it only allows a limited amount of emails, so you should [configure your own mail server](/email/providers).
```
Comment (santiagoaguiar) [inline articles/multifactor-authentication/factors/email.md]:
```suggestion
[Auth0 provides a test email provider](/email) but it only allows a limited amount of emails, so you should [configure your own email provider](/email/providers).
```

File: articles/multifactor-authentication/factors/email.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+Note that Email is not true 'Multi-factor Authentication' as it does not represent a different factor than the password. It does not represent 'something I have' or 'something I am', but rather just another 'something I know' (the email password). It is also weaker than other factors, in that it's only as secure as the email itself (e.g. is it encrypted end-to-end?).
+
+## End-user experience
+
+After the login step, users will be presented with the most secure enabled factor. If they select 'Other login options', and then pick Email, they will be sent an email with a six-digit code that they will need to enter to complete the authentication flow.
+
+![Email End User 1](/media/articles/multifactor-authentication/mfa-email.png)
+
+After Email MFA is enabled, all new or existing users from Database Connections that do not have verified emails will get prompted to verify their emails after their log in. They will get a code in ther email they'll need to enter in the login page to continue.
```
Comment (santiagoaguiar) [inline articles/multifactor-authentication/factors/email.md]:
```suggestion
After Email MFA is enabled, all new or existing users from Database Connections that do not have verified emails will get prompted to verify their emails after they log in. They will get a code in their email they'll need to enter in the login page to continue.
```

**Review by santiagoaguiar — DISMISSED** [review]
Looks good!

File: articles/multifactor-authentication/factors/email.md (hunk 1 above)
Comment (joseluisdiaz) [reply articles/multifactor-authentication/factors/email.md]:
```suggestion
To set up Email, you need to enable the Email factor in the Dashboard.
```

**Review by joseluisdiaz — DISMISSED** [review]
Looks good! :-)

**Review by jeffreylees — DISMISSED** [review]

**Review by jeffreylees — APPROVED** [review]

**Pull request merged**